warehouse/static/sass/blocks/_package-header.scss

```diff
@@ -114,6 +114,7 @@
       border-right: 1px dotted $transparent-white;
       border-top: 1px dotted $transparent-white;
       border-bottom: 1px dotted $transparent-white;
+      border-left: 1px;
```

Comment:
Had to add this as changing `a` to `button` was creating extra white border to left as shown below
![clipboard](https://user-images.githubusercontent.com/5843007/38497546-1844fd92-3c1f-11e8-9e78-0a7a6fda5a50.png)